runelite-client/src/main/java/net/runelite/client/plugins/hiscore/HiscorePlugin.java

```diff
@@ -201,6 +206,31 @@ else if (BEFORE_OPTIONS.contains(option))
 		}
 	}
 
+	@Subscribe
+	public void onLookupMenuClicked(PlayerMenuOptionClicked event)
```

Comment:
I know you didn't write this code originally, but change the method name to `onPlayerMenuOptionClicked`.